compiler/z/codegen/UnaryEvaluator.cpp

```diff
@@ -310,21 +310,22 @@ OMR::Z::TreeEvaluator::dsqrtEvaluator(TR::Node * node, TR::CodeGenerator * cg)
    {
    TR::Node * firstChild = node->getFirstChild();
    TR::Register * targetRegister = NULL;
-   TR::Register * opRegister = cg->evaluate(firstChild);
 
-   if (cg->canClobberNodesRegister(firstChild))
+   //See whether to use SQDB or SQDBR depending on how many times it is referenced
```

Comment:
Space after `//` needed.

Reply:
I actually don't think this comment is very useful as it doesn't really tell us anything that reading the if statement below does. I would vote to just remove it.